akvo/rsr/signals.py

```diff
@@ -272,6 +272,11 @@ def employment_post_save(sender, **kwargs):
                     employers__organisation=organisation,
```

Comment:
You could also rewrite this as `employers__organisation__in=[organisation, organisation.content_owner]`.